cli/pkg/app/cli.go

```diff
@@ -13,7 +13,7 @@ import (
 
 type CLI struct {
 	Exec             ExecCmd             `cmd:"" help:"import data in directory" default:"withargs"`
-	Load             LoadCmd             `cmd:"" help:"load data from stdin into directory"`
+	Publish          PublishCmd          `cmd:"" help:"load data from stdin into directory"`
```

Comment:
A we make the `exec` command help text more clear that it is a pipeline of F-T-P fetch, transform, publish

Reply:
Also, the `default:withargs` part of the help text is meaningless as there is no context what that means or implies

Reply:
Updated the `exec` help text to include a short description that it does fetch transform and load.

`default:withargs` is used so that users don't have to explicitly use `exec` on the cli
e.g.
`ds-load exec okta -c <config.yaml>` can be written `ds-load okta -c <config.yaml>`

Reply:
Should text refer to `publish` instead of `load`?
So:
`cmd:"" help:"import data in directory by running fetch, transform and publish" default:"withargs"`

The comment about `withargs` is that it is unclear to a user what means or does! Is there a real use-case where I would use `ds-load` without this option?

Reply:
If I understand correctly `default:"withargs"` is how we tell kong that exec is the [default command](https://github.com/alecthomas/kong#supported-tags).
The help string (i.e. `ds-load --help`) doesn't contain the string "withargs". It's just a signal to kong that exec should be used if no other command is specified and that the command-line arguments should be passed to ExecCmd and not to the top-level CLI.
I would suggest we mention the fact that exec is the default command in the help string.

Reply:
I misread this, my bad, apologies